cellxgene_schema_cli/setup.py

```diff
@@ -15,15 +15,6 @@
     install_requires=requirements,
     python_requires=">=3.10",
     packages=["cellxgene_schema"],
-    package_dir={"cellxgene_schema": "cellxgene_schema"},
-    package_data={
-        "cellxgene_schema": [
-            "gencode_files/*gz",
-            "migrate_files/*json",
-            "schema_definitions/*yaml",
-            "migrate_files/*.txt",
-        ]
-    },
```

Comment:
i believe we need this, or else non-py files won't be included in the package

Reply:
nvm, we have a MANIFEST file already so this should be fine to remove

Reply:
https://github.com/chanzuckerberg/single-cell-curation/blob/main/cellxgene_schema_cli/MANIFEST.in

we should add to this manifest, however, `migrate_files/*json` and `migrate_files/*txt`. this directory contains files used by lattice for the migrate script, as needed